### PR TITLE
CORE-856 duration in kibana as a number (ms)

### PR DESCRIPTION
**Pull request by ehudc**

### CODE REVIEW

**Comment by jayhahn** [comment]
I assume you tested this with an actual service?

**Comment by ehudc** [comment]
good point, I'll test it with accounts as an example

**Comment by ehudc** [comment]
@jayhahn yup:

    accounts_1    | time="2017-06-30T19:32:09Z" level=error msg="rpc endpoint GetAccountByEmail failed" company_uuid= duration=2 email= error="pq: invalid input syntax for uuid: """ request_id=23543992-9361-4055-b754-79e70d2e9613 

that "2" is in milliseconds

File: grpc/interceptor/logger_test.go [hunk 1]
```diff
@@ -18,9 +18,9 @@ import (
 // LogLine is only used to parse log lines from interceptors making it
 // easier to test
 type logLine struct {
-	Msg       string `json:"msg"`
-	RequestID string `json:"request_id"`
-	Duration  string `json:"duration"`
+	Msg       string      `json:"msg"`
+	RequestID string      `json:"request_id"`
+	Duration  json.Number `json:"duration,Number"`
```
Comment (bobbytables) [inline grpc/interceptor/logger_test.go]: Maybe we should add a test to this line to check that a duration was included while we're here: https://github.com/namely/mjolnir/pull/11/files#diff-a4ec4a083b9c6bc6004ca2e26846896fL49

Comment (ehudc) [inline grpc/interceptor/logger_test.go]: look below it at https://github.com/namely/mjolnir/pull/11/files#diff-a4ec4a083b9c6bc6004ca2e26846896fR60 that checks duration was included in the "finished rpc" log

**Pull request merged**